### PR TITLE
Adds new integration [abacao/hass_wibeee]

### PR DESCRIPTION
**Pull request by abacao**
This PR adds the Wibeee component as a default component in HACS

### CODE REVIEW

**Comment by KTibow** [comment]
Please enable issues and update the README to promote HACS installation.

**Comment by abacao** [comment]
> Please enable issues and update the README to promote HACS installation.

Both done and done.
Do I need to do anything else to make this available from the installation without manual adding the custom repo?

**Comment by abacao** [comment]
Brand also added on the other repo.
How can I Re-run the tests?

**Comment by abacao** [comment]
Missing approval on the Brands side...

**Comment by abacao** [comment]
All checks have passed. Fork is only a warning!

**Comment by KTibow** [comment]
You should ask @ludeeus then, I'm not related to HACS, just very passionate about getting their repos added to the defaults.

**Comment by ludeeus** [comment]
As stated in the docs, that check can fail <https://hacs.xyz/docs/publish/include#check-fork> now the only thing that remains is review(s), the failure of that check is not blocking.

**Comment by luuuis** [comment]
I was looking into what it would take to add [luuuis/hass_wibeee](https://github.com/luuuis/hass_wibeee) as a default repository in HACS and I ran into this PR. For context, mine is a fork of [abacao/hass_wibeee](https://github.com/abacao/hass_wibeee) (this repository) which in turn is a fork of [juanjoSanz/hass_wibeee](https://github.com/juanjoSanz/hass_wibeee).

The forked repos are not active at this point and my fork has moved on significantly with improved reliability, multiple device support, energy support, a config flow for UI configuration and just keeping up with new HA versions in general ([luuuis/hass_wibeee - CHANGELOG](https://github.com/luuuis/hass_wibeee/blob/729f61b8bc3a4204851e17a36f1f60f10ba03cee/CHANGELOG.md)). It uses semver, has tagged/zipped releases and I am planning further improvements such as Local Push support.

Basically I think everyone would be best served if my fork replaces abacao/hass_wibeee in the HACS default repositories. @abacao what do you think, would you support this change? If you and @ludeeus both agree I will go through the checklist and prepare a new PR to replace the repository.

**Comment by abacao** [comment]
I will approve that changed. Go ahead and God speed 🖖
